packages/components/text/src/components/osds-text/osds-text.tsx

```diff
@@ -37,7 +37,7 @@ export class OsdsText implements OdsTextAttribute {
     return (
       <Host {...{
```

Comment:
would be nice to remove the unneccessary destructuring

Reply:
It's removed